### DISCUSSION TITLE
Feature Request: Expecting team discussions being supported on mobile app

### DISCUSSION THREAD

**Discussion by alanxone** [community/community#2919] [Mobile] [answered]
Discussion for open source repository shows big support to community, and I can see how useful it is.

However, it seems team discussion introduced in Nov, 2017 [press release](https://github.blog/2017-11-20-introducing-team-discussions/) has been forgotten for a while.

It can be a great tool integrated for team collaboration, to be like MS Teams for engineers, and there's another product aiming to the same goal, JetBrains Space.

It's okay to use browser to participate the discussion, but with mobile support, it will be more useful.

**Comment by AgrMayank** [comment]
Seconded. Integration with Github mobile app will make it hugely better and actually useful on the go!

**Comment by candyho** [comment, marked as answer]
Hey thanks for sharing your feedback! Team discussion is something in our backlog and we don't have a solid timeframe to support it on mobile yet. We'll keep monitoring the feedback and hopefully we can land on a plan to support team discussion on mobile in the near future.
